cfgov/cfgov/settings/base.py

```diff
@@ -398,28 +398,66 @@
 
 PRIVACY_EMAIL_TARGET = os.environ.get("PRIVACY_EMAIL_TARGET", "test@localhost")
 
-
 # Password Policies
-# cfpb_common password rules
-CFPB_COMMON_PASSWORD_RULES = [
-    [r".{12,}", "Minimum allowed length is 12 characters"],
-    [r"[A-Z]", "Password must include at least one capital letter"],
-    [r"[a-z]", "Password must include at least one lowercase letter"],
-    [r"[0-9]", "Password must include at least one digit"],
-    [
-        r"[@#$%&!]",
-        "Password must include at least one special character (@#$%&!)",
-    ],
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+        "OPTIONS": {
+            "min_length": 12,
+        },
+    },
+    {
+        "NAME": "login.password_validation.PasswordRegexValidator",
+        "OPTIONS": {
+            "regex": r"[A-Z]",
+            "message": "Password must include at least one capital letter. ",
```

Comment:
```suggestion
            "message": "Password must include at least one capital letter.",
```

Was there a reason for the extra spaces at the end of all of these error messages? Maybe it was from the previous implementation where these would each be combined into a single error?

Reply:
Yes, that's exactly why. I'll commit all of these suggestions!